Final Project Submission/Barden_FinalProject.cpp

```diff
@@ -0,0 +1,407 @@
+// Program Name: Final Project Game
+// Purpose: a text based game where your (hard) decisions influence the outcome of
+//          of the (hard) game, that actually isn't as hard as I intended due to me severely
+//          underestimating how much time it actually takes to write a (not so good) storyline
+// Author: Kyle Barden
+// Date Last Modified: April 19, 2019
+
+//***********
+// READ THIS: I'm gonna be breaking the 4th wall a lot as the narrator and the narrator is gonna
+//            be very rude and talk trash to you at points so don't take it personally, the game is supposed
+//            to make you upset, anyway that's all, here comes the narrator. Scences are also listed
+//            as functions in reverse chronological order (except intro) so start at the bottom of the program and read up.
+//***********
+
+#include <iostream>
+#include <cmath>
+#include <ctime>
+#include <cstdlib>
+#include <iomanip>
+
+using namespace std;
+//global variables used to track living status of party memebers 1 = alive, 0 = dead
+bool Jack = 1, Kirk = 1, Steve = 1, Sean = 1, Isabella = 1, Carl = 1, Player = 1;
+//used for player health, so you can't keep making dumb combat mistakes, (that was the intention going in but my story isn't as long as
+//I thought it would be so it's just there for show at this point
+int Health = 100;
+//used to keep track of how many bandits are left in the final fight, also kind of arbitrary because well, you'll see that it doesn't matter too much.
+int Bandits = 20;
+
+int RNG(int min, int max){
+  const int MIN = min; //set min random value
+  const int MAX = max; //set max random value
+  int Random; //random number
+  Random = (rand() % (MAX - MIN + 1) + MIN);
+
+  return Random;
+}
+
+char TwoChoices(char Choice, char c1, char c2, char c3, char c4){
```

Comment:
Good idea, but needs more explanation

Reply:
Also, maybe this should have the _initial_ `cin` and maybe a string for prompting the user